dumpling-overview.md

```diff
@@ -61,7 +30,36 @@ In the above commands, you need to modify `~/.bash_profile` to the path of your
 
 </CustomContent>
 
-## Export data from TiDB/MySQL
+For detailed usage of Dumpling, use the `--help` option or refer to [Option list of Dumpling](#option-list-of-dumpling).
+
+When using Dumpling, you need to execute the export command on a running cluster.
+
+<CustomContent platform="tidb">
+
+TiDB also provides other tools that you can choose to use as needed.
+
+- For backups of SST files (key-value pairs) or backups of incremental data that are not sensitive to latency, refer to [BR](/br/backup-and-restore-overview.md). 
+- For real-time backups of incremental data, refer to [TiCDC](/ticdc/ticdc-overview.md).
+- All exported data can be imported back to TiDB using [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md).
+
+</CustomContent>
+
+> **Note:**
+>
+> PingCAP previously maintained a fork of the [mydumper project](https://github.com/maxbube/mydumper) with enhancements specific to TiDB. This fork has since been replaced by [Dumpling](/dumpling-overview.md), which has been rewritten in Go, and supports more optimizations that are specific to TiDB. It is strongly recommended that you use Dumpling instead of mydumper.
+>
+> For the overview of Mydumper, refer to [v4.0 Mydumper documentation](https://docs.pingcap.com/tidb/v4.0/backup-and-restore-using-mydumper-lightning).
+
+- Support exporting data in multiple formats, including SQL and CSV.
+- Support the [table-filter](https://github.com/pingcap/tidb-tools/blob/master/pkg/table-filter/README.md) feature, which makes it easier to filter data.
+- Support exporting data to Amazon S3 cloud storage.
+- More optimizations are made for TiDB:
+    - Support configuring the memory limit of a single TiDB SQL statement.
+    - Support automatic adjustment of TiDB GC time for TiDB v4.0.0 and later versions.
+    - Use TiDB's hidden column `_tidb_rowid` to optimize the performance of concurrent data export from a single table.
+    - For TiDB, you can set the value of [`tidb_snapshot`](/read-historical-data.md#how-tidb-reads-data-from-history-versions) to specify the time point of the data backup. This ensures the consistency of the backup, instead of using `FLUSH TABLES WITH READ LOCK` to ensure the consistency.
+
+## Export data from TiDB or MySQL
 
 ### Required privileges
 
```

Comment:
Please add [this paragraph](https://github.com/pingcap/docs-cn/pull/7143/files#diff-d167a2ebb86d802c7a7279da3de5de87e8b616ed0a27755ef0027a8db570c147R61) to line 73:

```
本文假设在 `127.0.0.1:4000` 有一个 TiDB 实例，并且这个 TiDB 实例中有无密码的 root 用户。
```